Leaflet.draw-ellipse/leaflet.draw-ellipse-src.js

```diff
@@ -1,6 +1,6 @@
 /*
 	Leaflet.draw-ellipse, a plugin that adds ellipse drawing and editing tools to Leaflet powered maps.
-	(c) 2012-2013, Jacob Toye, Smartrak
+	(c) 2015-2016, Haley Quentmeyer
```

Comment:
hah I guess I should have watched my copy paste. I am, in fact, the author of this file but it is essentially a modification of draw-circle by Jacob.